vino.py

```diff
@@ -25,50 +29,32 @@ def getColorMap(self):
         :return: Return a colormap to visualise different samples on the plot.
         """
 
-        for i in range(0, len(self.numberOfEachSamples)):
-            if i == 0:
-                colormap = np.zeros(self.numberOfEachSamples[0])
-            else:
-                colormap = np.append(colormap, np.ones(self.numberOfEachSamples[i]) *5*i)
+        uniqueLabelsInOrder = sorted(set(self.labels))
+        possibleColorsInOrder = range(len(uniqueLabelsInOrder))
+        colors = {}
+        for identifier, color in zip(uniqueLabelsInOrder, possibleColorsInOrder):
+            colors[identifier] = color*5
+
+        colormap = []
+        for identifier in self.labels:
+            colormap.append(colors[identifier])
```

Comment:
J'ai l'impression que quelque chose ne fonctionne pas avec la fonction colormap. Le nombre de couleur sur nos graphiques dépasse le nombre de bouteilles. La fonction set() permettrait de simplement garder les 19 valeurs différentes mais les labels sont 0000-0001 etc..  Donc j'ai l'impression qu'on crée N couleurs différentes... J'ai testé une autre méthode et le couleurs semblaient fonctionner
```suggestion
        splitId = []
        for i in range(0, len(self.labels)):
            splitId.append(self.labels[i].split("-"))
        colormap = list(map(int, list(zip(*splitId))[0]))
        return np.array(colormap)
```